spec/system/bills/user_accesses_associated_bill_spec.rb

```diff
@@ -173,3 +149,139 @@
     end
   end
 end
+describe 'Usuário acessa uma fatura' do
+  it 'pendente' do
+    cpf = CPF.generate
+    data = Rails.root.join('spec/support/json/tenant.json').read
+    response = double('response', success?: true, body: data)
+    endpoint_route = "http://127.0.0.1:3000/api/v1/get_tenant_residence?registration_number=#{CPF.new(cpf).formatted}"
+    allow(Faraday).to receive(:get).with(endpoint_route).and_return(response)
+    residence = JSON.parse(response.body)['resident']['residence']
+    resident = JSON.parse(response.body)['resident']
+
+    condo_id = residence['condo_id']
+    condo_name = residence['condo_name']
+    number = residence['number']
+    unit_id = residence['id']
+
+    units = []
+    units << Unit.new(id: unit_id, area: 100, floor: 2, number: 3, unit_type_id: 1)
+
+    allow(Unit).to receive(:find).and_return(units.first)
+
+    create(:bill, condo_id:, unit_id:, issue_date: Time.zone.today.beginning_of_month,
+                  due_date: 10.days.from_now, total_value_cents: 500_00, status: :pending)
+
+    visit root_path
+    within 'form#get_tenant_bill' do
+      cpf.each_char { |char| find(:css, "input[id$='get_tenant_bill']").send_keys(char) }
+      click_on 'Buscar'
+    end
+
+    click_on number.to_s
+
+    expect(page).to have_content CPF.new(cpf).formatted
+    expect(page).to have_content resident['name']
+    expect(page).to have_content condo_name
+    expect(page).to have_content Time.zone.today.beginning_of_month.strftime('%d/%m/%Y')
+    expect(page).to have_content 10.days.from_now.strftime('%d/%m/%Y')
+    expect(page).to have_content number
+    expect(page).to have_content 'R$500,00'
+    expect(page).to have_content 'Taxa Condominial'.downcase
+    expect(page).to have_content 'Conta Compartilhada'.downcase
```

Comment:
Não seria mais facil escrever em downcase? e se esse for o padrao, por que nao esta em todos?